README.md

```diff
@@ -1,32 +1,73 @@
+
+## List of Existing IDS Deployment Scenarios
+
+| Name of the Deployment Scenario | Provider | Description | Used Components | 
+| -------- | -------- | -------- | -------- |
+| [Minimum Viable Data Space](Deployment-Scenarios/minimum-viable-data-space-top-ix.md) | [TOP-IX](https://www.top-ix.org/it/home/) | An example deployment of Minimum Viable Data Space (as defined by IDSA) | CA, DAPS and Connector |
+| [IDS Testbed Deployment Using Kubernetes Architecture](Deployment-Scenarios/minimum-viable-data-space-using-k8s.md) | [ATOS](https://atos.net/es/spain) | How to deploy an IDS-Testbed using Kubernetes | CA, DAPS and Connector |
 
 :rocket: [Minimum Viable Data Space](https://github.com/International-Data-Spaces-Association/IDS-testbed/blob/master/minimum-viable-data-space/MVDS.md) is also included as a deployment scenario, defined by IDSA Head Office. It is also a great place to start creating a deployment scenario.
 
-## What is a IDS Deployment Scenario? 
+## What is an IDS Deployment Scenario? 
+An IDS deployment scenario is a description of the steps and processes involved in delivering a component or a set of components to end-users. It outlines the various stages of deployment, from development and testing to release and optionally maintenance. The scenario typically includes information on the prerequisites (such as hardware and software requirements), configuration settings, and any necessary third-party integrations. It also includes details on the deployment environment, such as whether it will be installed on-premises or in the cloud, and how the system will be monitored and supported once it is live. A deployment scenario helps ensure that the deployment process is consistent, easily repeatable, reusable and efficient, and that the system is deployed in a way that meets the needs of end-users. 
+
 A deployment scenario can be considered as any implementation made with IDS-compliant components:
 - that allows sovereign data sharing (as defined by IDSA)
-- is built with the intention to resolve a problem, by means of secure and sovereign data sharing
+- is built with a clear purpose, to solve a problem
 - is adequately documented to enable others to follow the same path
 
+### Example: 
+![](images/IDS-Deployment-Scenarios-Patterns.png)
+As depicted on the image above, there are several ways to create a deployment scenario. As long as the implementation is made for a purpose and contains an IDS-compliant component, it can be considered as an IDS Deployment Scenario. 
+
 An envisaged flow for running an experiment with a IDS Deployment Scenario can be considered as depicted on the image below:
+![](images/creationprocess.png)
 
-![IDS Deployment Scenarios Process](images/ids-deployment-scenarios-process.png)
+### The Importance of Sharing Deployment Scenarios
```

Comment:
please check the capitalization: Sharing --> sharing ; Importance --> importance